modules/administration/pages/content-lifecycle-examples.adoc

```diff
@@ -216,11 +216,16 @@ All packages installed during the year match the current live patching kernel fi
 [#appstream-filters]
 == AppStream Filters
 
-If you are using {rhel}{nbsp}8 clients, you cannot perform package operations such as installing or upgrading directly from modular repositories like the {rhel} AppStream repository.
-You can use the AppStream filter to transform modular repositories into regular repositories.
+In a content lifecycle management project, you can use the AppStream filter to transform modular repositories into regular repositories.
 It does this by keeping the packages in the repository and stripping away the module metadata.
 The resulting repository can be used in {productname} in the same way as a regular repository.
 
+[NOTE]
+====
+Since {productname} 5.0, AppStream repositories are natively supported throughout the UI.
```

Comment:
Sooner or later, we probably can get rid of the version info and thus the ifeval variants.  Suggestion for now:

```suggestion

ifeval::[{uyuni-content} == true]
AppStream repositories are natively supported throughout the {webui}.
endif::[]

ifeval::[{suma-content} == true]
Since {productname} 5.0, AppStream repositories are natively supported throughout the {webui}.
endif::[]

```
Empty lines before and after ifevals are tricky.  I'm not whether the parser requires them here.